README.md

```diff
@@ -433,6 +438,56 @@ To test functionality try uploading file to allocation. You should get the follo
 Upload failed. this options for this file is not permitted for this allocation: 
 file_option_not_permitted.
 ```
+##### Add Blobber
+
+Use `add_blobber` flag with [update allocation](#update-allocation) command to add blobber to allocation.
```

Comment:
also please also add: 
"the new blobber will be added as a parity blobber"

please adjust the grammar accordingly